### ISSUE TITLE
Optimized FSR for low power GPUs

### ISSUE THREAD

**Issue by SeppNel** [Blinue/Magpie#607]
### Expected behavior 预期的功能

This is a more performant implementation of fsr with slightly lower quality, great for low power systems.
The developer claims the EASU pass now takes 1.8ms from 4.7ms (measured running in an iPhone12).
[Here](https://gist.github.com/atyuwen/78d6e810e6d0f7fd4aa6207d416f2eeb) is the code. You can read more about it [here](https://atyuwen.github.io/posts/optimizing-fsr/).

**Comment by hooke007** [comment]
I'm sure it is dup years? ago.

**Comment by Blinue** [comment]
> EASU is fast enough for any desktop GPU, unless you want it running on mobilephone.

I totally agree, the performance improvement comes at the cost of image quality. But porting FSR_EASU_Lite is not difficult, and it’s a good thing to give users more choices. Although it’s a bit late, let's provide an “optimized” version of FSR, just like Lossless Scaling.
